templates/helm/values.yaml.tpl

```diff
@@ -110,7 +110,7 @@ deletionPolicy: delete
 # controller reconciliation configurations
 reconcile:
   # The default duration, in seconds, to wait before resyncing desired state of custom resources.
-  defaultResyncPeriod: 0
+  defaultResyncPeriod: 36000 # 10 Hours (https://github.com/aws-controllers-k8s/runtime/blob/main/pkg/runtime/reconciler.go#L48-L51)
```

Comment:
I would leave out the link to the code. It's not relevant to most customers who simply consume the default values.